core/tests/performance_tests/base.py

```diff
@@ -35,10 +36,24 @@ def _record_page_metrics_from_cached_session(self, page_url):
         self.page_metrics = (
             self.data_fetcher.get_page_metrics_from_cached_session(page_url))
 
-    def _record_page_timings_for_url(self, page_url):
-        self.page_metrics = (
-            self.data_fetcher.get_page_timings_for_url(page_url))
+    def _record_page_timings_for_url(self, page_url, session_count=3):
```

Comment:
Yeah! I have a comment regarding the same here: https://github.com/oppia/oppia/pull/2139#discussion_r67848463